cmd/security-secrets-setup/README.md

```diff
@@ -1,16 +1,88 @@
+
+Clean up `/run/edgex/secrets` once done verification if necessary.
+
+## Executable Run [cache] subcommand
+
+Before cache subcommand is issued, make sure cache dir (in this example, it is */tmp/pki/cache*) is non-existing or empty.
+
+To run the **cache** subcommand in linux system, change directory to *cmd/security-secrets-setup/* and then run 
+
+```sh
+mkdir -p /tmp/pki/cache && export PKI_CACHE=/tmp/pki/cache && sudo -E ./security-secrets-setup cache
+```
+
+After successfully run, verify that there are Vault and Kong TLS assets generated and cached into cache dir */tmp/pki/cache* and also deployed from cache directory into the deployed directory */run/edgex/secrets* via `ls` command.
+
+One can also use some file diff tool to compare the TLS files like:
+
+```sh
+sudo diff -r /tmp/pki/cache/ca/ca.pem /run/edgex/secrets/ca/ca.pem
+```
+
+If compares with the whole parent directories between cache dir and deployed dir, one should only find the difference in generated **sentinel** files.
+
+```sh
+sudo diff -r /tmp/pki/cache/ /run/edgex/secrets/
+
+Only in /run/edgex/secrets/ca: .security-secrets-setup.complete
+Only in /run/edgex/secrets/edgex-kong: .security-secrets-setup.complete
+Only in /run/edgex/secrets/edgex-vault: .security-secrets-setup.complete
+```
+
+Clean up `/run/edgex/secrets` once done verification if necessary.
+
+## Executable Run [import] subcommand
+
+To run the **import** subcommand in linux system, change directory to *cmd/security-secrets-setup/* and then run 
+
+```sh
+mkdir -p /tmp/pki/cache && export PKI_CACHE=/tmp/pki/cache && sudo -E ./security-secrets-setup import
+```
+
+There are two cases can happen depends on the status of cache directory:
+
+1. Cache dir */tmp/pki/cache* is empty:
+
+    In this case, one should expect to see the execution is blocking and waiting for TLS assets to be drop into cache directory before they can be deployed into directory */run/edgex/secrets/*.
+
+2. Cache dir */tmp/pki/cache* has already loaded or cached with generated TLS assets before run import subcommand:
+
+    One should see the deploying TLS assets happening right away from cached directory */tmp/pki/cache* into deployed directory */run/edgex/secrets*.  One can verify that there are Vault and Kong TLS assets deployed from cache directory */tmp/pki/cache* onto the directory */run/edgex/secrets* via `ls` command.
+
+Clean up `/run/edgex/secrets` once done verification if necessary.
+
+## Docker Build
+
+Use Makefile in the base directory of this repository to build the docker image of security-secrets-setup:
 
 ```sh
 make docker_security_secrets_setup
 ```
 
 It should create a docker image with the name `edgexfoundry/docker-edgex-vault:<version>-dev` if sucessfully built.
 
-# Run
+## Docker Run
 
 To see what the TLS materials are generated inside the docker container, 
 one can run the built docker image above (assuming the version number is `1.1.0`) as:
```

Comment:
Docker Run section is still showing the legacy output - should add some text to note that.

Reply:
Right, that is the current way it is in docker image...this will get changed when the docker image split-ie. after the other issue #1716 is addressed.